jenkins/bootstrap.py

```diff
@@ -883,7 +883,10 @@ def bootstrap(args):
             start(gsutil, paths, started, node(), version, repos)
         success = False
         try:
-            call(job_script(job, use_json))
+            job_cmd = job_script(job, use_json)
+            if upload:
+                job_cmd += ['--gcs-artifacts-dir=%s' % paths.artifacts]
```

Comment:
Yes that's true. But this is the only level at which the gcs-path to upload is known and we need to pass this information down to the kubekins container.
Also in case of other tests the flag just remains unused. Do you suggest some other alternative?